### ISSUE TITLE
shouldn't this be "+"?

### ISSUE THREAD

**Issue by rouniuyizu** [DAMO-DI-ML/KDD2023-DCdetector#7]
I'm a bit lost with the following final loss calc... Shouldn't this be the sum of two branches?

i.e.
```python
loss = prior_loss + series_loss 
```
https://github.com/DAMO-DI-ML/KDD2023-DCdetector/blob/bb2c19f137261f1e32a3429cd91a58abde6c0170/solver.py#L168

**Comment by yyysjz1997** [comment]
> I'm a bit lost with the following final loss calc... Shouldn't this be the sum of two branches?
> 
> i.e.
> 
> ```python
> loss = prior_loss + series_loss 
> ```
> 
> https://github.com/DAMO-DI-ML/KDD2023-DCdetector/blob/bb2c19f137261f1e32a3429cd91a58abde6c0170/solver.py#L168

Thanks for you issue. This should be "-" here because of the similarity of comparing the two representations. You can replace this with "+" to see what effect it has on the results, especially for datasets with low anomaly ratios.😊

**Comment by rouniuyizu** [comment]
> > I'm a bit lost with the following final loss calc... Shouldn't this be the sum of two branches?
> > i.e.
> > ```python
> > loss = prior_loss + series_loss 
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > https://github.com/DAMO-DI-ML/KDD2023-DCdetector/blob/bb2c19f137261f1e32a3429cd91a58abde6c0170/solver.py#L168
> 
> Thanks for you issue. This should be "-" here because of the similarity of comparing the two representations. You can replace this with "+" to see what effect it has on the results, especially for datasets with low anomaly ratios.😊

Thank you!
